packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/TranscribedLines/TranscribedLines.js

```diff
@@ -32,27 +32,33 @@ class TranscribedLines extends React.Component {
       show: false
     }
 
+    this.createMark = this.createMark.bind(this)
     this.close = this.close.bind(this)
     this.showConsensus = this.showConsensus.bind(this)
   }
 
   createMark (line) {
-    const { activeTool, activeToolIndex, setActiveMark } = task
+    const { activeTool, activeToolIndex, setActiveMark } = this.props.task
     const [{ x: x1, y: y1 }, { x: x2, y: y2 }] = line.points
     const markSnapshot = { x1, y1, x2, y2, toolIndex: activeToolIndex }
 
     if (activeTool) {
       const mark = activeTool.createMark(markSnapshot)
       mark.finish()
       setActiveMark(mark)
+
+      let previousAnnotationValuesForEachMark = []
+      activeTool.tasks.forEach((task) => {
+        const previousAnnotationValuesForThisMark = {
+          id: mark.id,
+          taskKey: task.taskKey,
+          taskType: task.type,
+          values: line.textOptions
+        }
+        previousAnnotationValuesForEachMark.push(previousAnnotationValuesForThisMark)
+      })
+      mark.setSubTaskVisibility(true, this.ref.current, previousAnnotationValuesForEachMark)
```

Comment:
I think `this.ref.current` is the parent element containing all the lines, including the individual line we want? I think that's causing an issue with the subtask positioning. If I moved `ref={this.ref}` to the each ConsensusLine component the subtask positioning appeared to be what I expect, but have not tested thoroughly.

Reply:
I'm opened a separate issue for this and have noted this observation to it: https://github.com/zooniverse/front-end-monorepo/issues/1702#issuecomment-656712459